### ISSUE TITLE
spacing/offsets for 2.5.8 Target size minimum

### ISSUE THREAD

**Issue by jamieherrera** [w3c/wcag#3182]
Hi, clarification needed for 2.5.5 vs 2.5.8 on target size vs touchable area when it comes to the spacing/offsets exception. It appears that 2.5.8 is primarily concerned with the _distance_ between touchable targets, rather than putting a minimum width/height to the _size_ of the surface area of what can actually be touched while _secondarily_ reducing risk of overlap with other interactions. 

The spacing section in the Understanding document for 2.5.8 appears to imply that any combination of touchable (target) area plus spacing can equal 24 px to reach a minimum of 24 px. The example shows an icon that is 20 px + 4 px spacing. But I could easily see someone interpret that as using (or continuing the practice of) a very small touchable area such as the close button on ads, and add a sizeable spacing around it that does nothing when touched or clicked on, whether or not there are other nearby interactive elements. Technically that would meet the current 24 px draft recommendation, but is a generally frustrating user experience and seems to not make an interaction more accessible for the user types this SC is intended to help, namely people with motor skill disabilities. If the minimum was called out, such as 20 px, for the actual interaction, that would be so much clearer.

**Comment by patrickhlauke** [comment]
as per the note in the understanding:

> this Success Criterion defines a minimum size and, if this can't be met, a minimum spacing. It is still possible to have very small, and difficult to activate, targets and meet the requirements of this Success Criterion, provided that the targets don't have any adjacent targets that are too close. However, using larger target sizes will help many people use targets more easily. As a best practice it is recommended to at least meet the minimum size requirement of the Success Criterion, regardless of spacing. For important links/controls, consider aiming for the stricter [2.5.5 Target Size (Enhanced)](https://w3c.github.io/wcag/21/target-size-enhanced.html).

So yes, there is in effect no real minimum size. but avoiding small targets that are also too close to each other still helps avoid accidental activation of the wrong target, which was the compromise the working group came to since an absolute actual minimum size could not be agreed on.

**Comment by jamieherrera** [comment]
Hi Patrick!
So the main point of Target size is to not accidentally activate other interactions, NOT to have an element that is big enough to touch? The definition of target size is still quite vague. I'd suggest adding a definition for "touch" vs "target size", since these have been used interchangeably in several contexts I've been in but in 2.5.8 Target size seems to indicate the touchable area + spacing. 

In my conversations in several workplaces related to 2.5.5 the main reason the companies I've worked with have implemented it as a best practice despite being AAA is to have a number to apply in the design phase for the touch size of 44px / 48dp larger surface area on mobile apps. Overlap was an issue when icons were near each other, so I appreciate a smaller size option than 44, but I have a feeling that this spacing exception is going to trip up people, and almost seems to make the experience less accessible for situations where nearby interactions are not nearby, but the design is intentionally made to be a very small touchable area that users can't activate; in the current draft that would pass 2.5.8.

**Comment by jamieherrera** [comment]
Yes, I understand. So will the draft include clarification on target size?

Any pointer input is going to have issues with a small interactive area.
It’s just not that clear from an Understanding perspective what the term
target size means. Small ineractive areas are already unusable for many
people; I understand that introducing the AA minimum needs to have a
relatively reachable expectation.

My main request is to better define target size than the current
definition, as many interpret these SC as having a minimum interactive area
rather than a minimum area between two interactions.

Thank you!

**Comment by patrickhlauke** [comment]
target size is still target size, as in the interactive area. this SC defines the minimum target size, but then immediately steps back and allows for the spacing *exception* to that measurement. so unfortunately, the naming of the SC is perhaps a bit misleading (as the exception there kicks in to change the tack of the SC)

**Comment by jamieherrera** [comment]
Right, I’m asking for an update to the term itself.

**Comment by jamieherrera** [comment]
got it, no update to the term. Still confusing.
